modules/julia/tutorials/julia.markdown

```diff
@@ -0,0 +1,121 @@
+Introduction to Julia OpenCV Binding {#tutorial_julia}
+=======================================
+
+OpenCV
+------
+
+OpenCV (Open Source Computer Vision Library) is an open source computer vision and machine learning software library. OpenCV was built to provide a common infrastructure for computer vision applications and to accelerate the use of machine perception in the commercial products. Being a BSD-licensed product, OpenCV makes it easy for businesses to utilize and modify the code.
```

Comment:
Please, replace "Being a BSD-licensed product, ..." with "Distributed under permissive license, ...", since we are are now changing the license from BSD to Apache 2.0

Reply:
I took this from the OpenCV website. I believe that'll need a change as well.